docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+kubectl apply -f scalaradminutils.yaml
+```
+
+Verify pod status:
+
+```console
+kubectl get pods scalaradminutils
+```
+
+### Pause
+
+Use the following command to pause the Scalar DL service for few seconds
+
+```console
+kubectl exec -i -t scalaradminutils -- ./bin/scalar-admin -c pause -s <SRV_Service_URL>
+```
+Note:-
+* Cosmos DB and DynamoDB will create backup automatically.
+
+### Unpause
+
+Use the following command to unpause the Scalar DL service after at least 10 seconds
+
+```console
+kubectl exec -i -t scalaradminutils -- ./bin/scalar-admin -c unpause -s <SRV_Service_URL>
+```
+
+## Restore
+
+This section shows how to restore transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must use a middle value of the paused time as restore point.
+* You must restore Scalar Ledger and Auditor tables with the same restore point if you are using the ledger and audit services.
+
+### Cosmos DB
+
+* Follow the [Restore an Azure Cosmos DB account that uses continuous backup mode](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) document to restore.
+* Change the default consistency to STRONG after restoring the data.
+
+### DynamoDB
+
+1. Restore tables by specifying the time
+    
+    * On the Amazon DynamoDB console, you can restore the tables one by one. 
+    
+        A. Restore the PITR (Point In Time Restore) backup of scalar/auditor tables on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html#restoretabletopointintime_console).
+        
+        B. Create the backup of previously restored tables (A) using the [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Backup.Tutorial.html#backup_console).
+        
+        C. Delete all scalar/auditor tables except the previously restored tables (A).
+        
+        D. Restore the previously created backup (B) using the actual scalar/auditor table name (previously deleted tables (C)) on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Restore.Tutorial.html#restoretable_console).
+    
+    * On the AWS Command Line Interface, You can restore all tables using following shell script.
+        * Configure AWS CLI with the region of DynamoDB.
+        ```console
+        aws configure
+        ```
+        * Download the [Point In Time Restore](./script/restore_dynamo.sh) script.
+        * Execute the script with restore point as UNIX time (UTC) and all table names
+            * `time` must be without millisecond
+            * All tables except `scalardb.metadata` must be passed to the restore script
+        ```console
+        #Example
+        #For ledger restoration
+        ./restore_dynamo.sh --time 1613960773 coordinator.state scalar.asset scalar.asset_metadata scalar.certificate scalar.contract scalar.contract_class scalar.function
```

Comment:
As a guide, let's focus on describing the steps.
The tool should be more properly created and tested.

Reply:
Sorry for the inconvenience.
This section has been removed from this guide